using-timescaledb/continuous-aggregates.md

```diff
@@ -69,7 +69,7 @@ on or off at any time.
 
 >:TIP: Upgrading continuous aggregates that were created in a version earlier than
 TimescaleDB 1.7 to use real-time aggregates should also ALTER the view to
-set `timescaledb.materialized_only=false`.   Subsequent to this change, queries 
+set ```timescaledb.materialized_only=false```.   Subsequent to this change, queries 
```

Comment:
Don't think this is necessary

Reply:
It doesn't affect how it parses.  If you want it in a non-inline code block, you need the backticks on separate lines
> \`\`\`
code here
\`\`\`

Reply:
Looks fine inline though:
<img width="814" alt="Screen Shot 2020-04-22 at 12 14 39 PM" src="https://user-images.githubusercontent.com/12288273/80006442-e0cf6000-8492-11ea-87d7-555afeea1806.png">

Reply:
Oops, yes, I meant this to be single ticks not three ticks.